firmware/include/widgets/clockWidget.h

```diff
@@ -4,6 +4,35 @@
 #include <globalTime.h>
 #include <widget.h>
 
+#ifndef CLOCK_FONT
+#define CLOCK_FONT DSEG7
+#endif
+
+#ifndef CLOCK_FONT_SIZE
+#define CLOCK_FONT_SIZE 200
+#endif
+
+// Some fonts are not really centered, use this to compensate
+#ifndef CLOCK_OFFSET_X_DIGITS
+#define CLOCK_OFFSET_X_DIGITS -10
+#endif
+
+#ifndef CLOCK_OFFSET_X_COLON
+#define CLOCK_OFFSET_X_COLON 0
+#endif
+
+// Not all digits in DSEG7/14 are aligned identically, therefore we need to offset them
+#ifndef CLOCK_DIGITS_OFFSET
+#define CLOCK_DIGITS_OFFSET { {0, 0}, {1, -5}, {0, 0}, {1, 0}, {0, -5}, {0, 0}, {0, 0}, {0, -5}, {0, 0}, {0, 0} }
```

Comment:
I would assume this info is available in the font file as well.

Reply:
The clock with shadow is especially picky because we first draw a 8 (in bg color) and then the actual number (in fg color).
If the character has even the slightest offset, we are screwed.

And I could not find a way to calculate this offset with the OpenFontRender methods.

But, same as above, I would be glad if someone could figure it out.